src/main/java/com/food4thought/food4thought/controller/FoodController.java

```diff
@@ -1,18 +1,37 @@
 package com.food4thought.food4thought.controller;
 
 import com.food4thought.food4thought.model.Food;
+import com.food4thought.food4thought.service.FoodService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
-import java.util.ArrayList;
 import java.util.List;
 
+@RequiredArgsConstructor
 @RestController
 public class FoodController {
+    private final FoodService foodService;
 
-    @GetMapping
-    public List<Food> GetAllFood() {
-        //todo
-        return new ArrayList<>();
+    @GetMapping("/menu")
```

Comment:
same as before, this can be getFoods, and you can use @RequestMapping on the class with the /menu value.